server/chat/background/task.py

```
@@ -611,6 +611,13 @@ def run_background_chat(
             except Exception as e:
                 logger.error(f"[BackgroundChat] Failed to link session to incident: {e}")
         
+        # Hook: check if LLM call is allowed (billing, rate limiting, etc.)
+        from utils.hooks import get_hook
+        hook_allowed, hook_message = get_hook("before_llm_call")(None, user_id)
+        if not hook_allowed:
+            logger.warning(f"[BackgroundChat] Hook blocked for user {user_id}: {hook_message}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)